wordpress-mobile/WordPress-Android
# Remove "new" prefix from site creation classes and resources

Fixes #9051

Removes the "New" prefix from site creation classes/resources as the classes/resources related to the old site creation have been removed in #9931.

I haven't renamed string resources as they have already been exported to GlotPress and renaming them would remove all the translations. 

To test:
1. Walk through the site creation flow and make sure the app doesn't crash 

Update release notes:

- There are not user facing changes
